api/v1/helpers/verbs/doGetHierarchy.js

```diff
@@ -29,6 +29,32 @@ function doGetHierarchy(resultObj) {
   const params = resultObj.params;
   const depth = Number(params.depth.value);
   resultObj.dbStartTime = Date.now();
+
+  const fields = params.fields.value;
+  const filterFields = fields && fields.length;
+  const subjectTags = params.subjectTags.value;
+  const filterByTags = subjectTags && subjectTags.length;
```

Comment:
The fields filter is only applied to the top-level object. Is this correct? If it was applied to the entire hierarchy we would also need to make sure the other 3 filter fields are included (aspectTags, aspect, status).